src/mjolnir/ingest_transit.cc

```diff
@@ -570,8 +570,8 @@ void write_routes(Transit& tile, const tile_transit_info_t& tile_info, feed_cach
     auto currRoute = feed.get_route(tile_routeId);
 
     route->set_name(currRoute->route_short_name);
-    route->set_onestop_id(currRoute->route_id);
-    route->set_operated_by_onestop_id(currRoute->agency_id);
+    route->set_onestop_id(feed_route.first + "_" + currRoute->route_id);
+    route->set_operated_by_onestop_id(feed_route.first + "_" + currRoute->agency_id);
```

Comment:
I was about to describe the way we determine onestop_ids for the filtering, and realized we don't do that yet. But I also kinda forgot that we should actually add it in #4032 . shouldn't have it added it here, but now it happened  :sweat_smile: